lambdas/common/utils/errors.ts

```diff
@@ -123,8 +123,8 @@ export class JwtSignatureValidationError extends BaseError {
 }
 
 export class SessionNotFoundError extends BaseError {
-  constructor(public readonly id: string) {
-    super(`Could not find session item with id: ${id}`);
+  constructor(public readonly message: string) {
+    super(message);
     this.statusCode = 400; // check
```

Comment:
This `//check` comment is starting to follow the code, we should break the copy and paste pattern and remove it.